README.md

```diff
@@ -1,56 +1,160 @@
-## `Intl.RelativeTimeFormat` API Specification [draft]
+# `Intl.RelativeTimeFormat` API Specification [draft]
 
-### Status
+## Overview
+
+### Motivation
+
+Due to common use, relative time formatted values exist in a majority of websites and are available in the majority of the frameworks (e.g., React, via [react-intl](https://github.com/yahoo/react-intl) and [react-globalize](https://github.com/globalizejs/react-globalize); Ember, via [ember-intl](https://github.com/ember-intl/ember-intl)). Popular localization libraries like [momentjs](https://momentjs.com/), [formatjs](https://formatjs.io/), [globalizejs](http://globalizejs.com/), and [others](https://github.com/rxaviers/javascript-globalization) had implemented a formatting process for relative time values as well.
+
+It is highly probable that the majority of current relative time formatting implementations require a large portion of CLDR raw or compiled data to format relative time values. Bringing this into the platform will improve performance of the web and developer productivity as they no longer have to bring extra weight to format relative time values.
+
+### Usage
+
+```js
+// Create a relative time formatter in your locale.
+let rtf = new Intl.RelativeTimeFormat("en", {
+    style: "long", // "long" (default), "short", or "narrow"
+});
+
+// Format relative time using the day unit.
+rtf.format(
+  -1,
+  "day" // "year", "quarter", "month", "week", "day", "sun", "mon", "tue", "wed", "thu", "fri", "sat", "hour", "minute", or "second".
+);
+// > "yesterday"
+```
+
+### Implementation Status
 
 __Stage 1__
 
 Implementation Progress
 
- * Polyfill (in progress)
+- Polyfill (in progress)
 
 Backpointers
 
- * https://github.com/tc39/ecma402/issues/35
+- https://github.com/tc39/ecma402/issues/35
 
-### Authors
+#### Authors
 
- * Caridy Patiño (@caridy)
- * Eric Ferraiuolo (@ericf)
- * Zibi Braniecki (@zbraniecki)
+- Caridy Patiño (@caridy)
+- Eric Ferraiuolo (@ericf)
+- Zibi Braniecki (@zbraniecki)
+- Rafael Xavier (@rxaviers)
+- Daniel Ehrenberg (@littledan) 
 
-### Reviewers
+#### Reviewers
 
 TBD
 
-### Informative
+## Proposal
+
+`Intl.RelativeTimeFormat` is a low level API to facilitate libraries and frameworks to format relative time in a localized fashion by providing internationalized messages for date and time fields, using customary word or phrase when available.
+
+### Spec
+
+You can view the [spec text](spec/relativetimeformat.html) or rendered as [HTML](https://rawgit.com/tc39/intl-relative-time-spec/g/master/index.html).
+
+### Technical Design
```

Comment:
For the design, it would be nice to talk about why this isn't *more* low-level, as the committee asked. For example, we could just expose the object directly that the API is querying. The answer here, I think, is that this is already a pretty low-level library, it's nicely analogous to the other Intl APIs, and is well-suited to writing libraries on top (which might take different policies); maybe libraries could make use of the pattern itself for detailed user options or caching, but this is pretty speculative and marginal, cc @maggiepint.